packages/trader/src/Modules/Trading/Components/Elements/PayoutPerPoint/payout-per-point.scss

```diff
@@ -0,0 +1,82 @@
+.payout-per-point-input {
+    padding-bottom: 0rem;
+    .popover_wrapper_class {
+        margin-top: 0.3rem;
+    }
+    .actions-wrapper {
+        margin-top: 0.6rem;
+        display: flex;
+        justify-content: space-between;
+        .distance-to-current-spot {
+            display: flex;
+            .barrier-value {
+                margin-bottom: 0.8rem;
```

Comment:
can we use quill tokens in this file for this and other rem units :)?
all quill tokens are listed here: https://github.com/deriv-com/quill-tokens/blob/main/lib/styles/quill.css